legislators-social-media.yaml

```diff
@@ -5450,3 +5451,46 @@
     govtrack: 412704
   social:
     facebook: rogermarshallmd
+    twitter: RepMarshall
+- id:
+    bioguide: E000296
+    govtrack: 412677
+  social:
+    twitter: RepDwightEvans
+- id:
+    bioguide: R000609
+    govtrack: 412692
+  social:
+    facebook: RepRutherfordFL
+    twitter: RepRutherfordFL
```

Comment:
Yeah, another sweep picked up Rep. Rutherford, Dwight Evans, and Mike Johnson. Those are all included here.